apps/me/.env.local.example

```
@@ -0,0 +1,24 @@
+# =============================================================================
+# F3 Me — Local Development Environment (example)
+# =============================================================================
+# Don't edit this file directly. Run `pnpm env:generate` to auto-generate
+# apps/me/.env.local from GCP Secret Manager (project: f3-me-app-staging).
+#
+# If you need manual setup, copy this file to .env.local and fill in values.
+
+# Auth (F3 SSO — points to local auth server on :3004)
+OAUTH_CLIENT_ID=f3-me-local
+OAUTH_CLIENT_SECRET=<from-gcp: f3-me-app-staging/oauth-client-secret>
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)